source/upgrade.txt

```diff
@@ -52,6 +52,27 @@ changes between the current and upgrade versions. For example, if you
 are upgrading the driver from v4.0 to v4.5, address all breaking changes from
 the version after v4.0 including any listed under v4.5.
 
+.. _java-breaking-changes-v5.0:
+
+Version 5.0 Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Several changes were made to the ``ConnectionId`` class.
+  
+  The ``ConnectionId`` constructor now accepts a value of type ``long`` as its second
```

Comment:
I think these could be nested bullets for easier readability (applies to all nested paragraphs in this section)